lib/Dialect/RTL/RTLOps.cpp

```diff
@@ -1315,6 +1315,172 @@ void ReadInOutOp::build(OpBuilder &builder, OperationState &result,
   build(builder, result, resultType, input);
 }
 
+//===----------------------------------------------------------------------===//
+// StructCreateOp
+//===----------------------------------------------------------------------===//
+
+static ParseResult parseStructCreateOp(OpAsmParser &parser,
+                                       OperationState &result) {
+  llvm::SmallVector<OpAsmParser::OperandType, 4> operands;
+  Type declStructType;
```

Comment:
Plz declare this variable as StructType, `parseColonType` will then check to make sure it is a struct.